admin_ui/backend/api/agents.py

```
@@ -331,15 +383,27 @@ def get_agent(slug: str):
-                         (slug,)).fetchone()[0]
+    # Match both the slug and the original name (call_records store the raw
+    # context_name, e.g. "Tool_Example") so legacy calls are counted (cf. MED-A3).
+    names = tuple({slug, row.get("display_name")} - {None})
+    placeholders = ",".join("?" * len(names))
+    # LOW-CH2: guard like the sibling endpoints so a missing call_records table
+    # (file exists but engine never ran) degrades to zeros instead of a 500.
+    try:
+        with sqlite3.connect(f"file:{CALL_HISTORY_DB}?mode=ro", uri=True) as c:
+            calls = c.execute(
+                f"SELECT COUNT(*) FROM call_records WHERE context_name IN ({placeholders}) "
+                "AND start_time >= datetime('now','-30 days')", names).fetchone()[0]
```

Ruff (0.15.17)

[error] Possible SQL injection vector through string-based query construction

(S608)